tools/osbuilder/rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -36,7 +36,7 @@ EOF
 [intel-sgx]
 source=https://download.01.org/intel-sgx/sgx_repo/ubuntu
 suite=$UBUNTU_CODENAME
-packages=libtdx-attest=1.20\*
```

Comment:
Bump to 1.22, using 1.2\* can cause us issues.

Reply:
done!